lib/suppressions.cpp

```diff
@@ -85,10 +85,16 @@ std::string SuppressionList::parseFile(std::istream &istr)
         if (line.empty())
             continue;
 
+        std::string::size_type pos = 0;
+        while (pos < line.size() && std::isspace(line[pos]))
+            ++pos;
+        if (pos == line.size())
+            continue;
+
         // Skip comments
-        if (line.length() > 1 && line[0] == '#')
+        if (line[pos] == '#')
             continue;
-        if (line.length() >= 2 && line[0] == '/' && line[1] == '/')
+        if (pos < line.size() - 1 && line[pos] == '/' && line[pos + 1] == '/')
```

Comment:
in general I prefer `pos + 1 < line.size()` instead of `pos < line.size() - 1` it's more robust in case the string is empty. but well in this particular case we do know that it's not empty.